submission/submission-core/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/request/SubmissionRequestReleaser.kt

```diff
@@ -111,6 +119,54 @@ class SubmissionRequestReleaser(
         return releasedFile
     }
 
+    private suspend fun suppressRequest(
+        accNo: String,
+        request: SubmissionRequest,
+    ) {
+        val sub = request.submission
+        logger.info { "$accNo ${sub.owner} Started suppressing submission files over ${sub.storageMode}" }
+        suppressSubmissionFiles(sub)
+        logger.info { "$accNo ${sub.owner} Finished suppressing submission files over ${sub.storageMode}" }
+    }
+
+    private suspend fun suppressSubmissionFiles(sub: ExtSubmission) {
+        suspend fun suppressFile(reqFile: SubmissionRequestFile) {
+            when (val file = reqFile.file) {
+                is NfsFile -> {
+                    val released = reqFile.copy(file = suppress(sub, reqFile.index, file), status = SUPPRESSED)
+                    rqtService.updateRqtFile(released)
+                }
+
+                is FireFile -> {
+                    if (file.published) {
```

Comment:
how it be posible to have a released submission with not published files? even here after we un publish the file file will remain as `published = true` it looks to me that we do not keep this flag consistent as our source of truth is submission.

Reply:
I think this was coming from when we didn't update the status per file but at the request level so we added this in order to avoid making unnecessary calls to FIRE for files that were already published. Now that we're querying by status, this should no longer be needed.